.cicdtemplate/.github/workflows/deploy_staging_and_production_to_firebase_app_distribution.yml

```diff
@@ -38,6 +39,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
+      - name: Set up google-services.json on template-compose
+        env:
+          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
+          GOOGLE_SERVICES_JSON_STAGING: ${{ secrets.GOOGLE_SERVICES_JSON_STAGING }}
+        run: |
+          mkdir -p template-compose/app
+          echo $GOOGLE_SERVICES_JSON > template-compose/app/google-services.json
```

Comment:
@luongvo I don't think it applies to all flavors. As I tested on my side, if we have the google-services.json on the root path, and have another one in `staging` path, it would use the correct file which belongs to staging correctly. Or it might be overridden for staging later. 🤔  BTW, this path works tho. 😂

Reply:
@luongvo I see. I updated it already. 🙏🏻